### ISSUE TITLE
Offset Pokemon despawn time

### ISSUE THREAD

**Issue by Trildar** [pompobit/ScanGO#7]
I don't know if it's just me, but for some reason the Pokemon despawn timers always seem to show with 30 minutes added to it. (Compared to another scanner that seems accurate)

Something to do with timezones I would guess. I'm in Malaysia.

**Comment by pompobit** [comment]
I did some research and setting the UTC+8 Kuala Lumpur timezone leads to an issue where indeed the timer is 30 minutes off. Setting another UTC+8 timezone works as expected.
It seems that when that timezone is set, Android system reports an unix timestamp 30000 millis lower than expected.
I found this https://en.wikipedia.org/wiki/Time_in_Malaysia

>  At 2330 hrs local time of 31 December 1981, people in Peninsular Malaysia adjusted their clocks and watches ahead by 30 minutes to become 00:00 hours local time of 1 January 1982, to match the time in use in East Malaysia, which is UTC+08:00.

This could explain the bug. I'm searching for a fix
